### PR TITLE
completed css-flexbox

### PR DESCRIPTION
**Pull request by Logisticx**
Align Items
image

Display Flex
image

Flex Wrap
image

Justify Content
image

### CODE REVIEW

File: css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  gap: 30px;
+  width: calc(140% - 30px);
+  flex-direction: row;
+}
+
+.card-wrapper {
+  display: flex;
+  flex-wrap: wrap;
```
Comment (Zhiwutian) [inline css-flexbox/flex-wrap.html:22]: these should not be on the wrapper. the wrapper should have the flex-basis and a little padding

**Comment by Logisticx** [comment]
Updated with missing padding.

File: css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 30px;
+  width: calc(140% - 30px);
+  flex-direction: row;
```
Comment (Zhiwutian) [inline css-flexbox/flex-wrap.html:16]: these properties are not needed and should be removed

File: css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck{
+      display: flex;
+      flex-basis: 25%;
```
Comment (Zhiwutian) [inline css-flexbox/justify-content.html:13]: flex basis should be on the card wrapper, not the card deck

File: css-flexbox/align-items.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-direction: row;
+      flex-basis: calc(100% / 3);
```
Comment (Zhiwutian) [inline css-flexbox/align-items.html:14]: flex basis should be on the card wrapper, not the card container

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
some updates are needed

**Pull request merged**